### PR TITLE
sign up for help button

### PR DESCRIPTION
**Pull request by Kendiya242**
Sign up for help name change to Join eco-friends

### CODE REVIEW

File: src/pages/home/Home.jsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ function Home() {
         <h1 className={styles.anton}>and Connect People</h1>
       </section>
       <section className={styles.button}>
-          <button className={styles.signupButton} onClick={handleSignupClick}><p className={styles.center}>Sign up for help</p ></button>
+          <button className={styles.signupButton} onClick={handleSignupClick}><p className={styles.center}>Join eco - friends</p ></button>
```
Comment (dinaolmelak) [inline src/pages/home/Home.jsx:42]: can you change that to `eco-friends` instead?

**Review by dinaolmelak — CHANGES_REQUESTED** [review]

File: src/pages/home/Home.jsx (hunk 1 above)
Comment (Kendiya242) [reply src/pages/home/Home.jsx:42]: I will capitalized the E for eco-friends.

Comment (dinaolmelak) [reply src/pages/home/Home.jsx]: I guess that is fine with me

**Review by dinaolmelak — APPROVED** [review]

File: src/pages/home/Home.jsx (hunk 1 above)
Comment (Kendiya242) [reply src/pages/home/Home.jsx]: DONE

**Pull request merged**